relnotes/reverse.feature.md

```diff
@@ -0,0 +1,5 @@
+* `ocean.core.array.Iteration`
+
+  New module containing single function for now - `reverse`, which allows to
+  iterate given slice in reverse order without any allocations (similar to
+  built-in `foreach_reverse`).
```

Comment:
"it gets a job done" - same as library replacement
"worse perfectly where applicable" - worse than library replacement
"additional code complexity" - additional language feature to learn, library version is trivially simple

I don't see any pragmatism here, only "it worked good enough for me" argument.